types/wlr_input_timestamps_v1.c

```diff
@@ -0,0 +1,383 @@
+#include <assert.h>
+#include <inttypes.h>
+#include <stdlib.h>
+#include <util/signal.h>
+#include <wayland-server.h>
+#include <wayland-util.h>
+#include <wlr/types/wlr_input_timestamps_v1.h>
+#include <wlr/types/wlr_seat.h>
+#include <wlr/util/log.h>
+#include "input-timestamps-unstable-v1-protocol.h"
+
+#define INPUT_TIMESTAMPS_MANAGER_VERSION 1
+
+static const struct zwp_input_timestamps_manager_v1_interface manager_impl;
+static const struct zwp_input_timestamps_v1_interface impl;
+
+/**
+ * helper functions
+ */
+
+static struct wlr_input_timestamps_v1 *
+wlr_input_timestamps_v1_from_resource(struct wl_resource *resource) {
```

Comment:
Static functions shouldn't have the `wlr_` prefix. This is reserved for exported functions.